python/samples/getting_started/azure_functions/07_single_agent_orchestration_hitl/function_app.py

```diff
@@ -16,10 +16,9 @@
 
 import azure.durable_functions as df
```

Comment:
Why this import and in other samples?

Can you quickly run `isort` on all the samples to remove unused imports.